gapic-generator/templates/default/helpers/presenters/method_presenter.rb

```diff
@@ -59,6 +59,8 @@ def doc_description
       .each_line
       .map { |line| line.start_with?(" ") ? line[1..-1] : line }
       .join
+      .split("{").join("\\\\\\{") # The only safe way to replace with \ characters...
+      .split("}").join("\\}")
```

Comment:
These lines are duplicated across the 5 presenters in this PR. Any way to refactor?

Reply:
Yes, good point. I'll refactor.